_component_design/actions-menu.md

```diff
@@ -1,17 +1,110 @@
 ---
 title: Actions Menu
-parent: Editing
-secondary: Inputs and Controls
+parent: Inputs and Controls
+secondary: Editing
 layout: component
 category: Components
 usage: >
-  The actions menu is used anytime a group of actions can be performed on a specific object on the current page or the current page. By grouping these actions within the actions menu, it allows users to have a single location to trigger multiple types of actions.
-preview-image: components/preview-help.svg
-status: unknown
-resource: false
-last-modified: 2017-08-17
+  The actions menu is used anytime a group of actions can be performed on a specific object on the current page. By grouping these actions within the actions menu, it allows users to have a single location to trigger multiple types of actions.
+preview-image: preview-images/actions-menu.svg
+status: stable
+resource: true
+last-modified: 2017-11-03
 ---
 
 {% include toc.html %}
 
-<!-- unknown -->
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- Use actions menus to consolidate multiple related actions into a single pop-over menu.
+
+### When not to use
+
+- If a table object has only a limited number of associated actions (such as create and delete), use table controls at the top of the table. For more information, see [tables]({{site.baseurl}}/components/tables.html) documentation.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Example actions menus from various triggers"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/actions-menu/actions-menu-hero.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+### Best practices
+
+- Links inside an actions-menu must always initiate an action related to the current page and must never be used simply for navigation. Actions can include the following: 
+
+	- Immediate actions on the current page. 
+	- Actions in [modals]({{site.baseurl}}/components/modals.html)
+	- Actions on a different page, or the start of a related workflow.
+
+- An actions-menu can be triggered by different elements including split buttons, buttons labeled 'actions', and cogs.
+
+- There are two different types of actions menus:
+
+  - Simple: Contains a list of links with no labels.
+  - Advanced: Contains list of links separated and organized through the use of labels.
+  
+- When using labels within an actionsmenu consider the following:
```

Comment:
s/`the following:`/`the following practices:`

Reply:
updated.